src/helpers/files.js

```diff
@@ -0,0 +1,49 @@
+import Vue from 'vue'
+import store from '../store/store.js'
```

Comment:
I'm a bit unsure whether it's worth pulling in the full Collectives store just for using `GET_COLLECTIVES_FOLDER` from it (see below) or whether it's better to duplicate a bit of code and reimplement `GET_COLLECTIVES_FOLDER` in order to not load the Collectives store each time the Files app is loaded :thinking:

Reply:
Yep, I'd say it rather makes sense to inject an initial state value when you load the additional files app script and just use that value then.

Reply:
Ok, agreed. I moved the logic to get the `collectivesFolder` to `helpers/files.js`.